ui/apps/platform/cypress/support/e2e.js

```diff
@@ -29,6 +29,7 @@ Cypress.on('scrolled', ($el) => {
 Cypress.on(
     'uncaught:exception',
     (err) =>
+        !err.message.includes('ResizeObserver loop completed') &&
```

Comment:
This seems to be cropping up for me now when local e2e tests visit the dashboard. Possible that a newer version of Chrome has changed the error message from the line below here.